### DISCUSSION TITLE
Alternative to Infima library. Is there a way to use any other react UI or css libraries?

### DISCUSSION THREAD

**Discussion by vivekVells** [facebook/docusaurus#8131] [Q&A]
Hi Team, 

Trying to find an alternative to [Infima](https://infima.dev/docs/components/alert). Unable to use any libraries as the docusaurs styling gets messed up. tried tailwindcss, antd, bootstrap. nothing worked. 

The objective is to re-use such libraries so that we don't have to spend too much time creating or styling components like text, toggle switch and so on. 

[Infima](https://infima.dev/docs/components/alert) looks good; but looks like it is at alpha stage and have very minimal components. 

Any thoughts?

**Comment by vivekVells** [comment]
Looks like the classnames conflict between docusaurus and any css libraries. could probably prepend the class names with libraries name (eg: if using [antd design](https://ant.design/), all css classes related to it can be prefixed with antd-<whatever_class_name>). have to figure out how to do this. 

but, is there any simpler way to achieve the objective in question description?

**Comment by slorber** [comment]
better discussed in https://github.com/facebook/docusaurus/issues/6032
